applications/StructuralMechanicsApplication/tests/adjoint_sensitivity_analysis_tests/adjoint_shell_structure_3d3n/linear_shell_test_nodal_disp_adjoint_parameters.json

```diff
@@ -12,6 +9,11 @@
         "solver_settings"                  : {
         "solver_type"                  : "adjoint_static",
         "analysis_type"                : "linear",
+        "model_part_name" : "Structure",
+        "domain_size"     : 3,
+        "time_stepping"                      : {
+            "time_step" : 1.0
```

Comment:
@armingeiser @MFusseder FYI I would use a timestep > 1.0 in order to avoid problems with numerics => suggested: 1.1